aggregate/decennial_census/census_2000_PUMS.py

```diff
@@ -0,0 +1,143 @@
+"""This script takes the xlsx files Erica from DCP Population sent us imports them, 
+cleans the column names to match our column schema naming conventions, filters out 
+unnecessary columns (the Housing Security and Quality Indicators 
+[Total Occupied Units, Owner Occupied, Renter Occupied and the corresponding racial 
+breakdowns]). 
+"""
+
+import pandas as pd
+from aggregate.PUMS.count_PUMS_demographics import PUMSCountDemographics
+from utils.PUMA_helpers import clean_PUMAs, census_races
+from internal_review.set_internal_review_file import set_internal_review_files
+from aggregate.aggregation_helpers import order_aggregated_columns, get_category
+
+demo_suffix = {
+    ## Rename the demographic race columns with wiki conventions
+    "_a": "_anh_",
+    "_b": "_bnh_",
+    "_h": "_hsp_",
+    #   "o00": "00_onh", No other non hispanic in excel file
+    "_w": "_wnh_",
+}
+
+
+def load_dec_2000_demographic_pop_demo():
+    df = pd.read_excel(
+        "./resources/decennial_census_data/EDDT_Census2000PUMS.xlsx",
+        skiprows=1,
+        dtype={"GeoID": str},
+    )
+    df = df.replace(
+        {
+            "GeoID": {
+                "Bronx": "BX",
+                "Brooklyn": "BK",
+                "Manhattan": "MN",
+                "Queens": "QN",
+                "Staten Island": "SI",
+                "NYC": "citywide",
+            }
+        }
+    )
+    df.set_index("GeoID", inplace=True)
+    return df
+
+
+def filter_to_demo_indicators(df):
+    """Remove columns that don't pretain to demographic indicators (Occupied units,
+    renter occupied units, owner occupied units which come 200 ACS PUMS Data but are
+    Housing Security and Quality category indicators"""
+    df = df.drop(
+        df.filter(regex="P25pl|LTHS|HSGrd|SClgA|BchD|Occ|OOcc|ROcc").columns, axis=1
+    )
+    return df
+
+
+def remove_duplicate_cols(df):
+    """Excel spreadsheet has some duplicate columns that Erica used for calculations
+    - keep columns ending with Z for now"""
```

Comment:
Noting that we're always keeping the "Z" column

Reply:
nice catch, removed that line in docstring